Xpirit-Vsts-Release-Terraform/terraform.ps1

```diff
@@ -134,9 +134,9 @@ function Initialize-Terraform
 
     if (-not ([string]::IsNullOrEmpty($additionalArguments)))
     {
-        $arguments = $remoteStateArguments + "$($extraArguments.Trim()) $($additionalArguments.Trim())"
```

Comment:
this broke because of the extra space you removed in the earlier commit. The extra space in the extraArguments should be removed to make it more clear